src/components/Form/FormProvider.js

```diff
@@ -239,6 +239,8 @@ function FormProvider({validate, shouldValidateOnBlur, shouldValidateOnChange, c
                 onSubmit={submit}
                 inputRefs={inputRefs}
                 errors={errors}
+                enabledWhenOffline={enabledWhenOffline}
+                formState={formState}
```

Comment:
The `formState` contains the `isLoading` prop, which indicates whether or not the form is loading after submitting. When I was testing, I noticed the button loading state wasn't being updated, which was due to the `formState` not being properly forwarded.

Reply:
In this case I wonder what would be the desired way of using the `formState`, because I see that it is taken from onyx in both: `FormProvider` and `FormWrapper`. After the suggested change we'll have two `formState` props overriding each other - one passed as a direct prop, the second taken from onyx. I think it may cause some problems, what do you think?

Reply:
Ah, I didn't notice the formState coming from Onyx. I guess I was just seeing some odd behaviour with the loading state when I was testing. I'll update and run some more tests.

Reply:
Updated. Not sure why I was seeing issues with loading earlier, but I cleared my browser caches and re-logged in just to make sure. Everything tests ok now.